File: 1041.cs
using System;
class URI {
    static void Main(string[] args) { 
        String [] X = Console.ReadLine().Split(' ');
        double n1 = double.Parse(X[0]);
        double n2 = double.Parse(X[1]);
        if(n1 > 0 && n2 > 0){
            Console.WriteLine("Q1");
        }
        if(n1 < 0 && n2 > 0){
            Console.WriteLine("Q2");
        }
        if(n1 < 0 && n2 < 0){
            Console.WriteLine("Q3");
        }
        if(n1 > 0 && n2 < 0){
            Console.WriteLine("Q4");
        }
        if(n1 == 0 && n2 != 0){
            Console.WriteLine("Eixo Y");
        }
        if(n1 != 0 && n2 == 0){
            Console.WriteLine("Eixo X");
        }
        if(n1 == 0 && n2 == 0){
            Console.WriteLine("Origem");
        }    
    }
}

File: 1041.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    double n1, n2;
    cin >> n1 >> n2;
    if(n1 > 0 and n2 > 0){
        cout << "Q1" << endl;
    }
    if(n1 < 0 and n2 > 0){
        cout << "Q2" << endl;
    }
    if(n1 < 0 and n2 < 0){
        cout << "Q3" << endl;
    }
    if(n1 > 0 and n2 < 0){
        cout << "Q4" << endl;
    }
    if(n1 == 0 and n2 != 0){
        cout << "Eixo Y" << endl;
    }
    if(n1 != 0 and n2 == 0){
        cout << "Eixo X" << endl;
    }
    if(n1 == 0 and n2 == 0){
        cout << "Origem" << endl;
    }
    return 0;
}
